ifbcat_api/management/commands/load_tools.py

```diff
@@ -81,7 +81,7 @@ def handle(self, *args, **options):
                             )
                             tool_type.save()
                             tool_type_list.append(tool_type)
-                            logger.debug(f"Type, {tool_keyword}, has been saved.")
+                            # logger.debug(f"Type, {tool_keyword}, has been saved.")
```

Comment:
I think it should be `tool_type` instead of `tool_keyword`, not removing the log

Reply:
Done!